README.md

```diff
@@ -72,10 +72,11 @@ TypeScript 2.0.
 5. Use your grammar in TypeScript
 
     ```typescript
-    import { ANTLRInputStream, CommonTokenStream } from 'antlr4ts';
+    import { CodePointCharStream, CodePointBuffer, CommonTokenStream } from 'antlr4ts';
 
     // Create the lexer and parser
-    let inputStream = new ANTLRInputStream("text");
+    let buffer = Buffer.from("text");
+    let inputStream = CodePointCharStream.fromBuffer(new CodePointBuffer(buffer, buffer.length));
```

Comment:
I also found this confusing. My IDE's recommended replacement was `const inputStream = CharStreams.fromString(items)`

Reply:
Yeah you're right, that one is even better.